maquetado/scss/tech.scss

```diff
@@ -54,3 +50,22 @@ $tech-logo-size: 100px;
   text-align: center;
   width: 130px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .technology {
+    border-top: 1px solid $green;
+  }
+
+  .technologies {
+    display: inline;
+  }
+
+  .wrap {
+    display: inline;
+  }
+
+  .first-card {
+    border: none;
```

Comment:
just to learn, instead of using this, add a pseudo-element to technolgy, something like nth-child or first-child